cmd/app-mesh-controller/root.go

```diff
@@ -150,3 +161,19 @@ func getConfig() (controllerConfig, error) {
 		},
 	}, nil
 }
+
+// addKlogFlags adds flags from k8s.io/klog
+func addKlogFlags(fs *flag.FlagSet) {
```

Comment:
This was borrowed from the kubernetes `globalflag` package: https://github.com/kubernetes/component-base/blob/kubernetes-1.15.0/cli/globalflag/globalflags.go#L41-L54